src/ray/object_manager/object_manager.cc

```diff
@@ -609,6 +609,8 @@ bool ObjectManager::ReceiveObjectChunk(const NodeID &node_id,
     // have to check again here because the pull manager runs in a different
     // thread and the object may have been deactivated right before creating
     // the chunk.
+    RAY_LOG(INFO) << "Aborting object creation because it is no longer actively pulled: "
```

Comment:
Remove?

Reply:
I'm trying to clarify the aborting object creation error message, by moving it from `ObjectBufferPool::AbortCreate()` to the respective callsites with hopefully more accurate description of the reason for aborting. LMK if this can be worded differently!

Reply:
Ah got it. I think it's okay for now, but in the future we should actually be pretty careful about moving INFO messages around since they can potentially suppress or trigger spurious messages.

Reply:
Sounds good, I reverted the info log changes for now.